README.adoc

```diff
@@ -46,6 +46,37 @@ gem install rouge
 ----
 ====
 
+== A Living Document
+
+Per the comment above, this guide is a work in progress - some rules are simply lacking thorough examples, but some things in the Minitest world change week by week or month by month.
```

Comment:
I intentionally didn't copy this part, as Minitest has been quite stable compared to RSpec and I don't think much has changed in years. :-)

Reply:
Removed it. I don't know why added it in the first place. Thanks